Activate the SQL_REMOVE_DELETED code.
Only applied if this is requested at port build time.

--- a/config.h.in
+++ b/config.h.in
@@ -371,3 +371,5 @@
 
 /* Define to `int' if <sys/types.h> doesn't define. */
 #undef uid_t
+
+#define ENABLE_SQL_REMOVE_DELETED
